optashift-employee-rostering-webapp/pom.xml

```diff
@@ -44,6 +44,110 @@
       <artifactId>logback-classic</artifactId>
       <scope>runtime</scope>
     </dependency>
+    
+     <!-- Testing -->
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <version>2.10.0</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
All version declerations should be in dependencyManagement in the root pom.

Reply:
I would normally agree to this, but arquillian is really picky about what dependencies you are allowed to have. Many of these dependencies are out-dated and are only there since arquilian requires them. Updating them to the one the other modules use breaks the test, and I don't want the other module using the out-dated dependency. I'll try to make it clearer.